tools/jdbc/src/main/java/org/duckdb/DuckDBNative.java

```diff
@@ -83,15 +83,15 @@ public class DuckDBNative {
 
 	protected static native String duckdb_jdbc_get_catalog(ByteBuffer conn_ref);
 
-	// returns stmt_ref result reference object
```

Comment:
Can you undo the changes to the JDBC source files? Adding the h prefix is just going to cause confusion

Reply:
Sure! Sorry, I did that by accident